paparazzi-gradle-plugin/src/main/java/app/cash/paparazzi/gradle/PrepareResourcesTask.kt

```diff
@@ -98,9 +111,9 @@ public abstract class PrepareResourcesTask : DefaultTask() {
       resourcePackageNames = resourcePackageNames,
       projectResourceDirs = projectResourceDirs.get(),
       moduleResourceDirs = moduleResourceDirs.get(),
-      aarExplodedDirs = aarExplodedDirs.relativize(gradleUserHomeDirectory),
+      aarExplodedDirs = aarExplodedResourcesBaseDirs.get(),
```

Comment:
per the above suggestion, this would become:
```suggestion
      aarExplodedDirs = aarExplodedDirs.get(),
```